Tiltfile

```diff
@@ -449,3 +450,8 @@ if localnet_config["faucet"]["enabled"]:
             "8080:8080",
         ],
     )
+
+
+### IBC relayer(s) & alt-chain node(s)
+load("./tiltfiles/ibc.tilt", "check_and_load_ibc")
+check_and_load_ibc(chart_prefix, localnet_config["ibc"])
```

Comment:
seems like the default configuration for ibc is missing (I realize it was moved to `tiltfiles/defaults.Tiltfile` recently, maybe that's why?)